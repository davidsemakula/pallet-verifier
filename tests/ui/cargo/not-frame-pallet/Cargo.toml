[package]
name = "not-frame-pallet"
version = "0.1.0"
edition = "2021"

[dependencies]
