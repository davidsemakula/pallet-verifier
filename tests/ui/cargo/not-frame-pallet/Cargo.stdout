Searching for FRAME pallet definition ...
